token-lending/program/src/state/reserve.rs

```diff
@@ -381,6 +382,8 @@ pub struct ReserveLiquidity {
     pub borrowed_amount_wads: Decimal,
     /// Reserve liquidity cumulative borrow rate
     pub cumulative_borrow_rate_wads: Decimal,
+    /// Reserve accumulated protocol fees
+    pub accumulated_protocol_fees_wads: Decimal,
```

Comment:
can we use same term as above - `cumulative_protocol_fees_wad`

Reply:
it means something different though